client/src/api/api.js

```diff
@@ -1,11 +1,18 @@
 import instance from './axios-config'
 
 export const getSampleHome = async () => {
-  const requestString = '/home'
+  const requestString = '/user'
   const res = await instance.get(requestString)
 
   if (!res?.data?.success) throw new Error(res?.data?.message)
```

Comment:
what's our stance on having both the condition and the statement in the same line?